lib/plugin.js

```diff
@@ -104,6 +105,7 @@ export default (ctx, inject) => {
   // Defaults
   const defaults = {
     retry: <%= parseInt(options.retry) %>,
+    timeout: <%= parseInt(options.timeout) %>,
```

Comment:
For two timeouts, this can be `process.server ? <%= parseInt(options.serverTimeout) %> : <%= parseInt(options.clientTimeout) %>`